src/conclusion/conclusion.tex

```diff
@@ -2,3 +2,59 @@
+The latest step in chess has been computer chess engines which reached beyond human skill with Deep Blue.
+Engines are now widely used for analysis, but the authors of this paper, have found that they also have immense
+potential for teaching chess.
+
+The saying ``Once a student, now the master'' shows that learning from masters of their craft is nothing new, but
+teaching in chess usually comes in the form of a coach.
+Even if this approach is one of the most effective teaching tools it has its drawbacks.
+Coaching costs money and needs to be planned to a specific time.
+This excludes people from pursuing this teaching method for other cheaper or more accessible solutions, and this is
+where engines can be the answer.
+
+State-of-the-art engine implementation in web apps do not allow online multiplayer integration, which disallows the
+social aspects of chess while learning with engines.
+In this project the authors have come to the conclusion that this interaction between two chess players is very
+important both for the learning experience, but also the enjoyment in learning.
+To further improve the accessibility making the solution as a web app would also allow people to play any time of day
+one could want
+
+In terms of target audience simply because of how chess is written and talked about, the authors' solution,
+ChessTeacher, does not cater to the completely novice players.
+These are players who do not know how or when specific moves can be made.
+The audience ChessTeacher will cater most to are beginners to intermediates, as they understand the rules and have begun
+learning patterns, which is also one aspect chess engines are superior to humans.
+
+While finishing the prototype user tests were made to understand how feasible the idea behind the project was, and also
+how good the implementation is.
+Although the reception of ChessTeacher was mixed, most of the critical feedback was related to the project only being
+a prototype and not what an eventual finished ChessTeacher would look like.
+This makes it certain that the approach has enough merit to be expanded and evolved further in future iterations.
+
+
+An interesting future for this approach to teaching chess would be testing how far up the ranking this approach is
+valid, as engines are the best chess players they should in theory also be the best teachers for chess, but this might
+not be practical.
+
+The solution for this project is not perfect though and one important aspect of coaching has been overlooked in
+ChessTeacher.
+This is the way the feedback is presented to the players, as currently you are only presented with the moves and no
+context to why this move is good.
+The main limiting factor for this, is that chess engines are not made to communicate context of moves, but only find
+good moves.
+This means it cannot explain why it chose the move, and this step of the board analysis is just as critical as the
+actual move the engines suggest.
```

Comment:
Maybe change to "... just as critical for the learning of the users as the ..." to make sure focus is on learning